### PR TITLE
Magnolia tuning

### PR DESCRIPTION
**Pull request by KadenKreisler**
Magnolia practice day changes

### CODE REVIEW

File: src/main/cpp/subsystems/LimelightSubsystem.cpp [hunk 1]
```diff
@@ -143,9 +146,10 @@ double LimelightSubsystem::GetApriltagShooterTheta(double dist, double angleTrim
     if(dist != 0.0)
     {
         frc::SmartDashboard::PutNumber("Distance AprilTag", dist);
-        // return -0.2351* pow((dist+angleTrim),3) + 4.38 * pow((dist+angleTrim), 2) - 29 * (dist+angleTrim) + 89.64;
-        return -0.2238* pow((dist+angleTrim),3) + 4.1431 * pow((dist+angleTrim), 2) - 27.38 * (dist+angleTrim) + 89.093;
-        // y = -0.2238x3 + 4.1431x2 - 27.38x + 89.093
+        // return -0.2351* pow((dist+angleTrim),3) + 4.38 * pow((dist+angleTrim), 2) - 29 * (dist+angleTrim) + 89.64; //initial curve not using the PID shooter 
+        // return -1.556* pow((dist+angleTrim),3) + 17.162 * pow((dist+angleTrim), 2) - 68 * (dist+angleTrim) + 127.2; //power of 3 magnolia
+        return 0.6042 * pow((dist+angleTrim),4) - 8.94 * pow((dist+angleTrim),3) + 49.7 * pow((dist+angleTrim),2) - 129.06 * (dist+angleTrim) + 168.18;
```
Comment (SgtRaven) [inline src/main/cpp/subsystems/LimelightSubsystem.cpp:151]: Is this going to work correctly with order of operations?

File: src/main/cpp/subsystems/SwerveModuleSubsystem.cpp [hunk 2]
```diff
@@ -58,6 +58,8 @@ SwerveModuleSubsystem::SwerveModuleSubsystem(
 
   frc::SmartDashboard::PutNumber("KFF Input " + std::to_string(m_driveMotor->GetDeviceId()), ModuleConstants::kFFModuleDriveController);
   frc::SmartDashboard::PutNumber("Wheel Offset " + std::to_string(m_turningMotor->GetDeviceId()), ModuleConstants::wheelOffset);
+
```
Comment (SgtRaven) [inline src/main/cpp/subsystems/SwerveModuleSubsystem.cpp:61]: Probably a good reason for it but the tenth second sleep functions sprinkled in?

**Review by SgtRaven — APPROVED** [review]
Approved pending a couple questions.

File: src/main/cpp/subsystems/LimelightSubsystem.cpp (hunk 1 above)
Comment (KadenKreisler) [reply src/main/cpp/subsystems/LimelightSubsystem.cpp:151]: I believe so, it's what we have been running all day

File: src/main/cpp/subsystems/SwerveModuleSubsystem.cpp (hunk 2 above)
Comment (KadenKreisler) [reply src/main/cpp/subsystems/SwerveModuleSubsystem.cpp:61]: To delay some can usage between setting up motors, hopefully for getting rid of the wheel randomly flipping issue.

**Pull request merged**